app/views.py

```diff
@@ -10,7 +10,9 @@ def index():
 @url.route('/query')
 def show_user_input():
     content = request.args.get('content')
-    return content or ("You've entered an incorrect/empty query. Please check and try again. You can try '/query?content=something relevant'", 400)
+    sms_from = request.args.get('from')
+    sms_to = request.args.get('to')
+    return 'Content: ' + content + ' SMS From: ' + sms_from + '  SMS To: ' + sms_to or ("You've entered an incorrect/empty query. Please check and try again. You can try '/query?content=something relevant'", 400)
```

Comment:
can you correct your error text?

Also, the `or` operator is not appropriate anymore, as the first operand will *never* be falsy